integration_test/gce/gce_testing.go

```diff
@@ -870,6 +870,17 @@ func UploadContent(ctx context.Context, logger *log.Logger, vm *VM, content io.R
 	return err
 }
 
+// RetrieveContent retrieves the file content from the the given file path from
+// the remote VM
+func RetrieveContent(ctx context.Context, logger *log.Logger, vm *VM, remotePath string) (content string, err error) {
+	if IsWindows(vm.ImageSpec) {
+		out, err := RunRemotely(ctx, logger, vm, fmt.Sprintf("Get-Content -Path '%s' -Raw", remotePath))
+		return out.Stdout, err
+	}
+	out, err := RunRemotely(ctx, logger, vm, fmt.Sprintf("sudo cat "+remotePath))
```

Comment:
[optional][nit] Here `fmt.Sprintf` is not necessary if we are concatenating the strings. I suggest to use only one, formatting with `Sprintf` or string concatenation.

Reply:
Done!